makefile.shade

```diff
@@ -574,9 +574,6 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(WIN10_SDK_LIB) && Directory.GetFiles
             {
                 var extraBinaries = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
                 {
-                    "System.Private.DataContractSerialization.dll",
-                    "System.Private.Networking.dll",
-                    "System.Private.ServiceModel.dll",
                     "System.Runtime.Serialization.Primitives.dll"
```

Comment:
Should we follow up with the serialization team? My guess is we don't want any of these binaries in the runtime if they aren't necessary.